api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -151,3 +169,84 @@ message Telemetry {
   //          returned.
   bytes frame_header = 6;
 }
+
+//----------------------------------------------------------------------------------------------
+// Scheduling APIs.
+//----------------------------------------------------------------------------------------------
+
+// Request for the `ListGroundstationPlans` method.
+message ListGroundstationPlansRequest {
+  // The ID of the groundstation to list plans for. The ID can be found on the StellarStation
+  // Console page for the groundstation.
+  string groundstation_id = 1;
+
+  // The start time of the range of plans to list. Only plans with an Acquisition of Signal (AOS)
+  // at or after this time will be returned. It is an error for the duration between `aos_after` and
+  // `aos_before` to be longer than 31 days.
+  google.protobuf.Timestamp aos_after = 2;
```

Comment:
Can we add aos_after is inclusive?

Reply:
I added it as words (at or after), but added inclusive too for good measure.